content/id/docs/tasks/tools/kubeadm/high-availability.md

```diff
@@ -0,0 +1,364 @@
+
+      ```sh
+      sudo kubeadm init phase upload-certs --upload-certs
+      ```
+
+    - Kamu juga dapat menentukan `--certificate-key` _custom_ pada saat `init` yang nanti dapat digunakan pada saat `join`.
+      Untuk membuat kunci tersebut kamu dapat menggunakan perintah berikut:
+
+      ```sh
+      kubeadm alpha certs certificate-key
+      ```
+
+    {{< note >}}
+    Secret `kubeadm-certs` dan kunci dekripsi akan kadaluarsa setelah dua jam.
+    {{< /note >}}
+
+    {{< caution >}}
+    Seperti yang tertera pada keluaran perintah, kunci sertifikat memberikan akses ke data klaster yang bersifat sensitif, jaga kerahasiaannya!
+    {{< /caution >}}
+
+2.  Pasang _plugin_ CNI pilihanmu:
+    [Ikuti petunjuk berikut](/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
+    untuk menginstal penyedia CNI. Pastikan konfigurasinya corresponds dengan CIDR Pod yang ditentukan pada berkas konfigurasi kubeadm jika diterapkan.
+
+    Pada contoh berikut kami menggunakan Weave Net:
+
+    ```sh
+    kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
+    ```
+
+3.  Tulis perintah berikut dan saksikan Pod komponen-komponen _control plane_ mulai dinyalakan:
+
+    ```sh
+    kubectl get pod -n kube-system -w
+    ```
+
+### Langkah-langkah selanjutnya untuk Node _control plane_
+
+{{< note >}}
+Sejak kubeadm versi 1.15 kamu dapat menggabungkan beberapa Node _control-plane_ secara bersamaan.
+Pada versi sebelumnya, kamu harus menggabungkan Node _control plane_ baru secara berurutan, setelah
+Node pertama selesai diinisialisasi.
+{{< /note >}}
+
+Untuk setiap Node _control plane_ kamu harus:
+
+1.  Mengeksekusi perintah untuk bergabung yang sebelumnya diberikan pada keluaran `kubeadm init` pada Node pertama.
+    Perintah tersebut terlihat seperti ini:
+
+    ```sh
+    sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866 --control-plane --certificate-key f8902e114ef118304e561c3ecd4d0b543adc226b7a07f675f56564185ffe0c07
+    ```
+
+    - Opsi `--control-plane` menunjukkan `kubeadm join` untuk membuat _control plane_ baru.
+    - Opsi `--certificate-key ...` akan membuat sertifikat _control plane_ diunduh
+      dari Secret `kubeadm-certs` pada klaster dan didekripsi menggunakan kunci yang diberikan.
+
+## Node etcd eksternal
+
+Membangun sebuah klaster dengan Node etcd eksternal memiliki prosedur yang mirip dengan etcd bertumpuk
+dengan pengecualian yaitu kamu harus setup etcd terlebih dulu, dan kamu harus memberikan informasi etcd 
+pada berkas konfigurasi kubeadm.
+
+### Memasang klaster etcd
+
+1.  Ikuti [petunjuk berikut](/id/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) untuk membangun klaster etcd.
+
+2.  Lakukan pengaturan SSH seperti yang dijelaskan [di sini](#manual-certs).
```

Comment:
if you decide to remove the alias anchor here (https://github.com/kubernetes/website/pull/22669/files#diff-6f6c6f2a2b1a160e3bf2cac7899fe8c6R282), this needs change

Reply:
Yep, fixed in bc5c6c5.